assets/js/blocks/mini-cart/mini-cart-contents/inner-blocks/filled-mini-cart-contents-block/frontend.tsx

```diff
@@ -12,13 +17,31 @@ const FilledMiniCartContentsBlock = ( {
 	children,
 	className,
 }: FilledMiniCartContentsBlockProps ): JSX.Element | null => {
-	const { cartItems } = useStoreCart();
+	const { cartItems, cartItemErrors } = useStoreCart();
+
+	const { createErrorNotice } = useDispatch( 'core/notices' );
+
+	// Ensures any cart errors listed in the API response get shown.
+	useEffect( () => {
+		cartItemErrors.forEach( ( error ) => {
+			createErrorNotice( decodeEntities( error.message ), {
+				isDismissible: false,
```

Comment:
@gigitux it seems that something about this `isDismissible` setting isn't taking. I am now seeing two `X` close buttons - one for the message and one for the cart drawer (see screenshot below). I also notice that there is a `is-dismissible` class on the rendered element. I don't recall these being present in your previous commits so this appears to be a regression.

![CleanShot 2022-12-21 at 16 09 17](https://user-images.githubusercontent.com/481776/209003763-d981d8f9-ad79-4293-9936-ddc0a619d71b.png)

Reply:
Not sure, but it seems that the regression is caused by [this PR](https://github.com/woocommerce/woocommerce-blocks/pull/7711) that was merged 3 days ago. Looking into 👀

Reply:
Ok, as I suspected, #7711 introduces a regression that was fixed with #8018. I merged #8018 in this branch. Also, the PR introduced some styles that override the style that we added in this PR. I fixed this with [4416792](https://github.com/woocommerce/woocommerce-blocks/pull/7234/commits/441679268bbcb79bc10b7cc8e1abea6a3113b16c).